Cargo.toml

```diff
@@ -8,6 +8,7 @@ members = [
   "wp_derive_request_builder",
   "wp_serde_helper",
   "wp_uniffi_bindgen",
+  "wp_org_api",
```

Comment:
I think this name might be a bit confusing, because technically `wp_api` is the `.org` API. I am not sure if the API we are using to fetch the plugin directory has any other endpoints, so maybe we should just call it `wp_org_api_plugin_directory`?

Reply:
I don't think we'd want `_plugin_directory` in the library name, because there may be more we'll add, like the theme directory.

What do you think about `wordpress_org_api`?

Reply:
> What do you think about wordpress_org_api?

My 2¢ – this is a good compromise. There are other .org APIs that we'll want to support (the theme directory, for instance)